Gruntfile.js

```diff
@@ -181,9 +181,33 @@ module.exports = function (grunt) {
 	];
 
 	grunt.initConfig(gruntConfig);
-
 	grunt.registerTask('compiler', ['curl:compiler', 'unzip:compiler']);
-	grunt.registerTask('all', ['copy', 'concat', 'closure-compiler']);
+	grunt.registerTask('all', ['copy', 'concat', 'requirejs', 'closure-compiler']);
+
+	grunt.loadTasks('spec/tasks');
+
+	var browsers = grunt.option('browsers') || 'default';
+  grunt.registerTask('test',
```

Comment:
Mixing tabs and spaces? You are kidding, right :smile:? You are pretty good to keep a consistent style usually. I didn't expect you wouldn't care about this.